daemon/config/config.go

```diff
@@ -600,6 +609,61 @@ func findConfigurationConflicts(config map[string]interface{}, flags *pflag.Flag
 	return nil
 }
 
+// MergeSpecialConfig verifies configs and handles conflicts in changed values
+func MergeSpecialConfig(config *Config, configFile string, flags *pflag.FlagSet) (*Config, error) {
+
+	unmergedConfig, _err := New()
+	if _err != nil {
+		return nil, _err
+	}
+
+	b, err := os.ReadFile(configFile)
+	if err != nil {
+		b = []byte("")
+	}
+	err = json.Unmarshal(b, unmergedConfig)
```

Comment:
We need to handle all these errors cases.
If we can't unmarshal this then it needs to return.